src/ReactTags.js

```diff
@@ -286,6 +288,15 @@ class ReactTags extends Component {
     );
   }
 
+  handleKeyPress(e) {
```

Comment:
@przpl  thanks for the contribution. We already have a `handleInputChange` callback which gets triggered when input changes so this validation can be handled at parent component. IMO this validation should handled at parent level. What do you think ?